packages/ui-components/src/components/Switch/Switch.component.tsx

```diff
@@ -0,0 +1,310 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState, useEffect, useMemo, useId } from "react"
+
+import { Label } from "../Label/Label.component"
+import { Icon } from "../Icon/Icon.component"
+import { FormHint } from "../FormHint/FormHint.component"
+
+const switchBaseStyles = `
+    jn-rounded-full
+    jn-relative
+    jn-p-0
+    jn-leading-0
+    jn-border
+    jn-g-theme-default
+    focus:jn-outline-none
+    focus:jn-ring-2
+    focus:jn-ring-theme-focus
+    disabled:jn-opacity-50
+    disabled:jn-cursor-not-allowed
+`
+
+const borderBaseStyles = `
+    jn-border-theme-switch-default
+`
+
+const switchWrapperStyles = `
+    jn-flex
+    jn-flex-row
+    jn-items-center
+`
+
+const handleBaseStyles = `
+    jn-inline-block
+    jn-absolute
+    jn-top-[1px]
+    jn-rounded-full
+    jn-bg-theme-switch-handle
+    jn-border-theme-default
+`
+
+const handleOnStyles = `
+    jn-right-[1px] 
+    jn-bg-theme-switch-handle-checked
+`
+
+const handleOffStyles = `
+    jn-left-[1px]
+`
+
+const validBaseStyles = `
+    jn-border-theme-success
+`
+
+const invalidBaseStyles = `
+    jn-border-theme-error
+`
+
+const iconBaseStyles = `
+    jn-inline-block 
```

Comment:
is it like formating issue or I had something wrong here? :) I think we use 2 spaces right?

Reply:
Looks same as always to me? 🤪
Deprecated code: https://github.com/cloudoperators/juno/blob/main/packages/ui-components/src/components/Switch/Switch.component.js
Message PR: https://github.com/cloudoperators/juno/pull/511/files#diff-ee6a66d799330adbc1075e5714952d4380089bf042754385ff3cfaa1c248f5ee